### PR TITLE
requester: purge max request jobs

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

* Removes friction point while tracking how many HTTP outbound jobs are run in parallel. This is generally not needed because of the rate limiting systems in place and allows you to spawn pending routines which will queue and exceed this amount especially in the confines of rolling window data on start-up (e.g. fee synchronisation).

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/bybit/bybit_live_test.go [hunk 1]
```diff
@@ -11,7 +11,6 @@ import (
 	"testing"
```
Comment (gloriousCode) [inline exchanges/bybit/bybit_live_test.go:11]: Please remove in mock test

File: exchanges/binance/binance_mock_test.go [hunk 2]
```diff
@@ -12,7 +12,6 @@ import (
```
Comment (gloriousCode) [inline exchanges/binance/binance_mock_test.go:12]: Please remove in live test

**Review by gloriousCode — CHANGES_REQUESTED** [review]
I think it is reasonable for a user to be able to set their own limits/manage request frequency. I don't think the current system of throwing away the request is the best way of handling it. I can see room for improvement for such a system to manage burstiness, but if that ever happens - PRs welcome
So in that sense, I'm comfortable with the removal of this system

**Review by gloriousCode — APPROVED** [review]
tACK m8

**Review by thrasher- — APPROVED** [review]
tACK. This was originally intended to cap a person from spamming thousands of go routines and subsequent requests by accident but most of the important exchanges support a rate limiter now. I'd still say we should have some kind of limit on the ones which don't but can be addressed in a separate PR.

**Pull request merged**